hyppo/independence/hhg.py

```diff
@@ -44,6 +53,10 @@ class HHG(IndependenceTest):
         where ``x`` is the data matrix for which pairwise distances are
         calculated and ``**kwargs`` are extra arguements to send to your custom
         function.
+
+    fast : boolean, default: False
```

Comment:
call it auto to be consistent with other hyppo functions

Reply:
Changed.